mod/module_test.go

```diff
@@ -31,15 +31,12 @@ func TestConfigGitHubMode(t *testing.T) {
 }
 
 func TestConfigGoModulesMode(t *testing.T) {
-	fs := afero.NewOsFs()
+	fs := afero.NewMemMapFs()
```

Comment:
Awesome! No more go mod files appearing! Also, I think this means we can remove the 

`defer removeGomodFile` since we don't create any files that need to be cleaned up in the actual filesystem